draft-irtf-cfrg-hpke.md

```diff
@@ -388,11 +421,24 @@ Note that `GenerateKeyPair` can be implemented as `DeriveKeyPair(random(Nsk))`.
 The notation `pk(skX)`, depending on its use and the KEM and its
 implementation, is either the
 computation of the public key using the private key, or just syntax
-expressing the retrieval of the public key assuming it is stored along
+expressing the retrieval of the public key, assuming it is stored along
 with the private key object.
 
 The following two functions are defined to facilitate domain separation of
-KDF calls as well as context binding:
+KDF calls, as well as context binding:
+
+{::comment}
+[rfced] Please review the "type" attribute of each sourcecode element
+in the XML file to ensure correctness. If the current list of preferred
+values for "type" (https://www.rfc-editor.org/materials/sourcecode-types.txt)
+does not contain an applicable type, then feel free to let us know.
+
+In addition, review each artwork element. Specifically, should any artwork
+element be tagged as sourcecode or another element?
+
+Examples of usage of the artwork element from the documentation:
+ diagrams ("line art") and protocol unit diagrams
+{:/comment}
```

Comment:
I don't think there's any artwork in this document.  For the most part, the `type` attribute should be `pseudocode`, except for the test vectors, which should be `test-vectors`.